spec/definitions/provision.sh

```diff
@@ -0,0 +1,47 @@
+# unmanaged-files
```

Comment:
You may be right, but I'd defer this change for now. The script was only moved here and it's working fine in its current form.